src/executor/attach_op_execs_pass.cc

```diff
@@ -23,42 +23,85 @@ const OperatorProperty* OpPropGetOpProperty(const NodeAttrs& attrs);
 
 namespace exec {
 
-// stateful compute executor
-class StatefulComputeExecutor : public OpExecutor {
+// abstract OpExecutor which provides storage fallback procedure on
```

Comment:
Seems there are no un-implemented member functions. The word "abstract" maybe is misleading. Probably pointing out that this the common class inherited by `StatefulComputeExecutor` and `FComputeExecutor` is clearer.